### PR TITLE
Update etcd store with new client

### PR DESCRIPTION
**Pull request by albertrdixon**
Use the context aware etcd client

### CODE REVIEW

File: store/etcd.go [hunk 1]
```diff
@@ -58,10 +69,14 @@ func (e *EtcdStore) GetMultiMap(prefix string) (map[string]io.Reader, error) {
 // Set will set a single Etcd key to value with a ttl.  If the ttl is zero then
 // no ttl will be set.  Ttls in etcd are in seconds and must be at least 1
 func (e *EtcdStore) Set(key string, value []byte, ttl time.Duration) error {
-	var etcdTTL = uint64(ttl.Seconds())
-	e.mutex.Lock()
-	defer e.mutex.Unlock()
-	if _, err := e.client.Set(key, string(value[:]), etcdTTL); err != nil {
+	// var etcdTTL = uint64(ttl.Seconds())
```
Comment (soellman) [inline store/etcd.go]: Remove?

File: store/etcd.go [hunk 2]
```diff
@@ -6,13 +6,22 @@ import (
 	"sync"
 	"time"
 
-	"github.com/coreos/go-etcd/etcd"
+	"golang.org/x/net/context"
+
+	"github.com/coreos/etcd/client"
 )
 
+func EnableEtcdDebug() {
```
Comment (soellman) [inline store/etcd.go:15]: What's this for?

Comment (albertrdixon) [inline store/etcd.go:15]: Just debugging. If the consumer wants to enable something like a 'Trace' debug level they can turn on etcd debug logging.

Comment (soellman) [inline store/etcd.go:15]: Cool, good idea.

**Comment by soellman** [comment]
LGTM

**Pull request merged**